### PR TITLE
AirfRANS: LR sensitivity at T_max=10 golden (lr=5e-4/8e-4/1e-3)

### PR DESCRIPTION
**Pull request by tcapelle**
## Hypothesis

The AirfRANS golden config uses lr=7e-4, which was optimized when T_max=5 was the standard. PR #2823 revealed that T_max=10 dramatically improves convergence (0.001479 → 0.001095). With longer cosine cycles, the optimizer explores further before the LR drops — meaning the effective step-size distribution changes. The LR that was optimal at T_max=5 may not be optimal at T_max=10. This experiment sweeps the LR neighborhood around 7e-4 at the new T_max=10 golden setting to find the true optimum.

## Baseline

- **Primary metric:** `val_primary/surface_mse` (lower is better)
- **Current best (pending merge, PR #2823):** 0.001095 at 3L/256d + gc=1.0 + T_max=10 + lr=7e-4 + WD=1e-2
- **Previous merged best:** 0.001479 (PR #2771)

## Runs

Run all 3 simultaneously (1 GPU each). Report **best val_primary/surface_mse checkpoint** (NOT final epoch).

**Run 1 (CUDA 0): lr=5e-4 at T_max=10**
```bash
cd target/icml2026 && CUDA_VISIBLE_DEVICES=0 SENPAI_MAX_EPOCHS=9999 SENPAI_TIMEOUT_MINUTES=180 python train.py \
  --dataset airfrans --airfrans-task full --optimizer adamw \
  --lr 5e-4 --cosine-t-max 10 --grad-clip 1.0 --weight-decay 1e-2 \
  --no-use-ema --enable-fourier \
  --model-layers 3 --model-hidden-dim 256 --model-heads 4 \
  --epochs 999 \
  --wandb-group airfrans-3L256d-Tmax10-lr \
  --wandb-name airfrans-Tmax10-lr5e4
```

**Run 2 (CUDA 1): lr=8e-4 at T_max=10**
```bash
cd target/icml2026 && CUDA_VISIBLE_DEVICES=1 SENPAI_MAX_EPOCHS=9999 SENPAI_TIMEOUT_MINUTES=180 python train.py \
  --dataset airfrans --airfrans-task full --optimizer adamw \
  --lr 8e-4 --cosine-t-max 10 --grad-clip 1.0 --weight-decay 1e-2 \
  --no-use-ema --enable-fourier \
  --model-layers 3 --model-hidden-dim 256 --model-heads 4 \
  --epochs 999 \
  --wandb-group airfrans-3L256d-Tmax10-lr \
  --wandb-name airfrans-Tmax10-lr8e4
```

**Run 3 (CUDA 2): lr=1e-3 at T_max=10**
```bash
cd target/icml2026 && CUDA_VISIBLE_DEVICES=2 SENPAI_MAX_EPOCHS=9999 SENPAI_TIMEOUT_MINUTES=180 python train.py \
  --dataset airfrans --airfrans-task full --optimizer adamw \
  --lr 1e-3 --cosine-t-max 10 --grad-clip 1.0 --weight-decay 1e-2 \
  --no-use-ema --enable-fourier \
  --model-layers 3 --model-hidden-dim 256 --model-heads 4 \
  --epochs 999 \
  --wandb-group airfrans-3L256d-Tmax10-lr \
  --wandb-name airfrans-Tmax10-lr1e3
```

## Expected Outcome

With longer cosine cycles (T_max=10) vs T_max=5, the optimizer can afford a slightly higher peak LR since the valley recovery is deeper and the LR stays elevated longer per cycle. lr=8e-4 or lr=1e-3 may find deeper basins than 7e-4. Alternatively, lr=5e-4 could provide more stable trajectories. We expect at least one variant to beat 0.001095.

Report: best val_primary/surface_mse for each run, epoch of best, and gradient norm behavior (any divergence?).

### CODE REVIEW

**Comment by morganmcg1** [comment]
Closing this PR because the assigned student pod was retired during the DrivAerML and cross-dataset refocus on 2026-04-21. We are freeing capacity for the new relaunch wave, and this lane is no longer being actively pursued. Any useful results already logged here or in W&B remain part of the research record.
